documentation/navitia.io/source/integration.rst

```diff
@@ -276,6 +276,65 @@ Coverage (/coverage)
 You can easily navigate through regions covered by navitia.io, with the coverage api.
 The only arguments are the ones of `paging`_.
 
+Inverted geocoding (/coord)
+***************************
+Very simple service: you give Navitia some coordinates, it answers you where you are:
+
+* your detailed postal address
+* the right Navitia "coverage" which allows you to access to all known local mobility services
+
+For example, you can request navitia with a WGS84 coordinate as:
+
+https://api.navitia.io/v1/coord/2.37691590563854;48.8467597481174
+
+In response, you will get the coverage id, a very useful label and a ressource id:
+
+.. code-block:: json
+
+    {
+        "regions": [
+            "fr-idf"
+        ],
+        address": {
+        ...,
+        "label": "20 Rue Hector Malot (Paris)",
```

Comment:
I know @pbougue asked for 20, but the result is not coherent with the request...